backend/tracim_backend/models/context_models.py

```diff
@@ -931,6 +931,29 @@ def pdf_available(self) -> bool:
         else:
             return False
 
+    @property
+    def has_jpeg_preview(self) -> bool:
+        """
+        :return: bool about if jpeg version of content is available
+        """
+        if self.content.depot_file:
+            from tracim_backend.lib.core.content import ContentApi
+            content_api = ContentApi(
+                current_user=self._user,
+                session=self.dbsession,
+                config=self.config,
+                show_deleted=True,
+                show_archived=True,
+                show_active=True,
+                show_temporary=True,
+            )
+            return content_api.has_jpeg_preview(
+                self.content.revision_id,
+                file_extension=self.content.file_extension
+            )
+        else:
+            return False
+
```

Comment:
@inkhey when you have a "short case" and a "long case" (in terms of number of lines) it's easier to read a code written like this:


```
if not self.content.depot_file:
    return False

content_api = ContentApi(
    current_user=self._user,
    session=self.dbsession,
    config=self.config,
    show_deleted=True,
    show_archived=True,
    show_active=True,
    show_temporary=True,
)
return content_api.has_jpeg_preview(
    self.content.revision_id,
    file_extension=self.content.file_extension
)
```

instead of this:

```
if self.content.depot_file:
    from tracim_backend.lib.core.content import ContentApi
    content_api = ContentApi(
        current_user=self._user,
        session=self.dbsession,
        config=self.config,
        show_deleted=True,
        show_archived=True,
        show_active=True,
        show_temporary=True,
    )
    return content_api.has_jpeg_preview(
        self.content.revision_id,
        file_extension=self.content.file_extension
    )
else:
    return False
```

The reasons are:

- less indentation
- short case is immediate to read
- long case is long to read (but its the case in both ways to write the code)

Reply:
done.